crates/dekaf/Cargo.toml

```diff
@@ -15,25 +15,49 @@ doc = { path = "../doc" }
 gazette = { path = "../gazette" }
 labels = { path = "../labels" }
 ops = { path = "../ops" }
+flowctl = { path = "../flowctl" }
 proto-flow = { path = "../proto-flow" }
 proto-gazette = { path = "../proto-gazette" }
 simd-doc = { path = "../simd-doc" }
 
 anyhow = { workspace = true }
 axum = { workspace = true }
+axum-server = { workspace = true }
 axum-extra = { workspace = true }
 base64 = { workspace = true }
 bytes = { workspace = true }
-clap = { workspace = true }
+# TODO(jshearer): Upgrade every other usage of clap to v4 and change this to:
```

Comment:
Upgraded all usage of `clap` to v4 -- I had wanted to avoid scope creep of an already fairly big PR, but I see your point about there being no better time than right now to pay down tech debt 👍